zephyr/ra/ra_cfg/fsp_cfg/bsp/ra8d1/bsp_mcu_family_cfg.h

```diff
@@ -574,9 +574,7 @@
 #define BSP_CFG_ROM_REG_ARC_OEMBL1 (0U)
 #endif
 
-#ifndef BSP_CFG_DCACHE_ENABLED
-#define BSP_CFG_DCACHE_ENABLED (0)
-#endif
+#define BSP_CFG_DCACHE_ENABLED (CONFIG_DCACHE)
```

Comment:
If there is no special reason, please keep it as FSP defined.
```suggestion
#ifndef BSP_CFG_DCACHE_ENABLED
#define BSP_CFG_DCACHE_ENABLED (CONFIG_DCACHE)
#endif
```

Reply:
Sorry for the unnecessary change, I have updated it. Thank you